<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OsagoBlankDataForm</class>
 <widget class="QDialog" name="OsagoBlankDataForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сведения о собственнике ТС</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Страхователь</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QCheckBox" name="insurancerIsOwner">
          <property name="text">
           <string>Является собственником</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QDateEdit" name="birthday">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="dateInnLabel">
            <property name="text">
             <string>Дата рождения гражданина/
ИНН юридического лица</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="dateInnLineEdit"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="fioLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="insurancerNameLabel">
            <property name="text">
             <string>Поное наименование юридического лица/
Фамилия, имя отчество гражданина</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Юридическое лицо</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Документ, удостоверяющий личность</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="3">
              <widget class="QLineEdit" name="serialLineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Номер</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Серия</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="typeLineEdit"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Тип документа</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2" colspan="2">
              <widget class="QLineEdit" name="numberLineEdit"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Адрес</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Индекс</string>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLineEdit" name="country">
               <property name="minimumSize">
                <size>
                 <width>143</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="index"/>
             </item>
             <item row="0" column="2" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Государство, республика, край</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="state"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Район</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="street"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Улица</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Корпус</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="corp"/>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Населенный пункт</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLineEdit" name="city"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Дом</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Квартира</string>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QLineEdit" name="house"/>
             </item>
             <item row="3" column="4">
              <widget class="QLineEdit" name="flat"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="phoneLayout">
          <item>
           <widget class="QLabel" name="phoneLabel">
            <property name="text">
             <string>Контактный телефон</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="phone"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="prevPushButton">
         <property name="text">
          <string>Назад</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>198</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton">
         <property name="text">
          <string>Далее</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
